.docker/gitpod/mathlib/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This is the Dockerfile for `leanprover/mathlib:gitpod`.
+
+# gitpod doesn't support multiple FROM statements, (or rather, you can't copy from one to another)
+# so we just install everything in one go
```

Comment:
Perhaps
```suggestion
# gitpod doesn't support multiple FROM statements, (or rather, you can't copy from one to another)
# so we just install everything in one go.
# https://community.gitpod.io/t/docker-compose-support/1656/8
```
assuming I'm right in thinking that https://community.gitpod.io/t/docker-compose-support/1656/8 tracks this shortcoming

Reply:
I don't think that issue is talking about this issue. `docker-compose` is a whole extra level of complication beyond what we're doing here (it's for builds that produce multiple containers, that can interact with each other over a network at runtime). Multiple FROM statements is just a trick to reduce build sizes, but discards all but the final container. It seems that gitpod discards them a bit too early: they work, but you can't copy artifacts out of them into later stages. I couldn't find an issue tracking this.